packages/server/src/service/util.ts

```diff
@@ -59,19 +59,27 @@ export const safeReadFileSync = (
  *
  * @param d exported for testing purposes, defaults to now when not present
  *
- * Based on: https://stackoverflow.com/a/6117889/9611230
+ * Based on: https://github.com/commenthol/weeknumber/blob/master/src/index.js
```

Comment:
commenthol/weeknumber doesn't appear to have type definitions, just adapted their algorithm here.